showcase/app/styles/_globals.scss

```diff
@@ -168,6 +168,61 @@ body {
   }
 }
 
+// special classes for headings with anchor links
+
+.shw-page-heading-scroll-margin-top {
+  scroll-margin-top: 24px;
+}
+
+.shw-page-heading-link {
+  position: relative;
+  display: inline-block;
+  width: 24px;
+  height: 1em;
+  min-height: 24px;
+  margin-top: 0.15em;
+  margin-left: -24px;
+  vertical-align: text-top;
+  outline: none;
+
+  &::after {
+    position: absolute;
+    top: 50%;
+    left: -2px;
+    width: 24px;
+    height: 24px;
+    background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='16' height='16' fill='none' viewBox='0 0 16 16'%3E%3Cg fill='%23666666'%3E%3Cpath d='M11.141 1a3.893 3.893 0 00-2.738 1.093L7.314 3.175A.75.75 0 008.372 4.24l1.077-1.07a2.393 2.393 0 013.384 3.382l-1.881 1.88a2.393 2.393 0 01-3.608-.258.75.75 0 00-1.202.899 3.893 3.893 0 005.87.42l1.886-1.886.01-.009A3.893 3.893 0 0011.14 1z'/%3E%3Cpath d='M7.019 5.365a3.893 3.893 0 00-3.032 1.13L2.102 8.382l-.01.01a3.893 3.893 0 005.505 5.504l1.084-1.084a.75.75 0 00-1.06-1.06l-1.07 1.07a2.393 2.393 0 01-3.384-3.384l1.881-1.88a2.393 2.393 0 013.609.258.75.75 0 101.2-.899A3.893 3.893 0 007.02 5.365z'/%3E%3C/g%3E%3C/svg%3E");
+    background-repeat: no-repeat;
+    background-position: center center;
+    background-size: 16px 16px;
+    border-radius: 3px;
+    transform: translateY(-50%);
+    opacity: 0;
+    content: "";
+
+    @media screen and (width <= 500px) {
+      left: 0;
+    }
+  }
+
+  :hover > & {
+    &::after {
+      opacity: 1;
+    }
+  }
+
+  &:focus-visible {
+    &::after {
+      background-color: aliceblue;
+      border: 2px dashed blue;
```

Comment:
Are these colors temporary for testing or would you want to replace them with more "official" colors?